framework/components/ecl-files/ecl-files.config.js

```diff
@@ -1,6 +1,91 @@
 module.exports = {
   title: 'Files',
   label: 'Files',
-  status: 'planned',
+  status: 'ready',
   tags: ['molecule'],
+  collated: false,
+  default: 'default',
+  variants: [
+    {
+      name: 'default',
+      label: 'Default file download',
+      context: {
+        title: 'File title example',
+        language: 'English',
+        meta: '213.25 kB - PDF - 4 pages',
+        buttonLabel: 'Download',
+        icon: 'file',
+      },
+    },
+    {
+      name: 'translations',
+      label: 'File download with translations',
+      context: {
+        variant: 'translations',
+        title: 'File title example',
+        language: 'English',
+        meta: '213.25 kB - PDF - 4 pages',
+        buttonLabel: 'Download',
+        icon: 'file',
+        translations: [
+          { title: 'Titre du fichier', meta: '228.84 kB - PDF - 4 pages' },
+          { title: 'Dateititel', meta: '232.12 kB - PDF - 4 pages' },
+          { title: 'Файл Заглавие', meta: '257.54 kB - PDF - 4 pages' },
+        ],
+        translationsLabel: 'Available languages (3)',
+        translationsTooltip: 'Click to see translations',
+        _demo: {
+          scripts: `document.addEventListener('DOMContentLoaded', function () {
+            ECL.initExpandables('#translations-expand-button');
+          });`,
+        },
+      },
+    },
+    {
+      name: 'links',
+      label: 'Links to files',
+      context: {
+        variant: 'links',
+        links: [
+          { title: 'First file', icon: 'file', href: '#', type: 'pdf' },
```

Comment:
I personally find the previous file names a bit more user-friendly, i.e. `Generic external file link`, `Link to a .pdf file`, etc.